mavenPublish.gradle

```diff
@@ -26,94 +26,75 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+apply plugin: 'java-library'
```

Comment:
This could be moved to the parent `build.gradle`

Reply:
no, there is no parent build for this one.

Reply:
With parent I meant the section in `build.gradle` where we have `apply from: rootProject.file('gradle/mavenPublish.gradle')`

Reply:
ok, done